configure.ac

```diff
@@ -617,6 +617,14 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <unistd.h>]],
  [ AC_MSG_RESULT(no)]
 )
 
+AC_MSG_CHECKING(for getentropy rand)
```

Comment:
nit: `for getentropy rand` sounds after we are looking for a new identifier/function. I recommend to use `for getentropy via random.h`.